libmattermost.c

```diff
@@ -415,6 +415,8 @@ mm_markdown_to_html(const gchar *markdown)
 		gchar **markdown_version_split = g_strsplit_set(	markdown_version, ". ", -1);
 		gchar *last_part;
 		guint i = 0;
+
+		printf("AA:%s:AA\n",markdown_version);

```

Comment:
But what is this?!! (Related to your other issue about the evil version of markdown?)